### PR TITLE
Contact profile component modified and test case added

### PR DESCRIPTION
**Pull request by Abro0058T**

### CODE REVIEW

**Comment by VishalZ123** [comment]
@Abro0058T 
Please add a detailed explanation of what your PR does.
I hope you understand this is not a good practice to submit PRs without any description.

File: screens/SavedSearches.tsx
Comment (ajaman190) [inline screens/SavedSearches.tsx:1]: Pull up the latest code, the latest code will include all the implementations of the saved search list component in the saved search screen, so the saved search list component is no longer needed, I mean the lookout latest collections and contacts screen implementation, you'll understand what to do.

File: screens/ContactProfile/ContactProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+          Status{'\n'}
+          <Text style={styles.text}>Valid contact</Text>
+        </Text>
+        <Text style={styles.subHeading}>
+          Collections{'\n'}
+          <Text style={styles.text}>Optin contacts</Text>
+        </Text>
+      </View>
+      <View style={styles.buttonContainer}>
+        <TouchableOpacity style={styles.button} onPress={() => navigate.navigate('MoreInfo')}>
```
Comment (ajaman190) [inline screens/ContactProfile/ContactProfile.tsx:68]: Create a single component for the button and then use it

File: screens/ContactProfile/ContactProfile.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+            source={require('../../assets/icon.png')}
+            style={styles.avatar}
+          />
+        </View>
+        <Text testID="userName" style={styles.nameText}>
+          {contact.name}
+        </Text>
+      </Pressable>
+      <View style={styles.subContainer} testID="mainContainer">
+        <Text style={styles.subHeading}>
```
Comment (ajaman190) [inline screens/ContactProfile/ContactProfile.tsx:46]: Create a component and resue it, similar to this
```sh
const NewComponent = (label, value) => {
       return (
          <Text style={styles.subHeading}>
             {label}
             <Text style={styles. Text}>{value}</Text>
           </Text>
        )
}

//usages
<NewComponent label={'Language\n'} value={'English'}/>
```

**Comment by mdshamoon** [comment]
Closing this since no updated have been made on this PR for a while

**Pull request closed without merging**